### ISSUE TITLE
tc not exposed by the composable useI18n()

### ISSUE THREAD

**Issue by Wuraim** [nuxt-modules/i18n#3136]
### Environment

------------------------------
- Operating System: Windows_NT
- Node Version:     v20.12.1
- Nuxt Version:     3.11.2
- CLI Version:      3.11.1
- Nitro Version:    2.9.6
- Package Manager:  npm@10.5.0
- Builder:          -
- User Config:      devtools, app, runtimeConfig, modules, css, alias, i18n, pwa, auth, piniaPersistedstate, primevue
- Runtime Modules:  @nuxtjs/i18n@8.5.1, @pinia/nuxt@0.5.1, @pinia-plugin-persistedstate/nuxt@1.2.1, @vite-pwa/nuxt@0.10.1, @sidebase/nuxt-auth@0.6.7, @nuxtjs/tailwindcss@6.12.1, nuxt-primevue@0.3.1
- Build Modules:    -
------------------------------

### Reproduction

You just have to notice that the keys 'tc' of the object returned by useI18n() is not declared.

### Describe the bug

When using the composable useI18n() to get access to the tc method for the pluralization, that function is not defined.
(Unlike the function t, which is defined)

![image](https://github.com/user-attachments/assets/b5974ec3-7e51-49e4-9e51-4f779b4822d5)

We are forced to use the $tc or other methods to do it

**Comment by BobbieGoede** [comment]
Did you update `vue-i18n` to v10? Nuxt i18n v8 does not support this version, but `vue-i18n` removed `tc` among other breaking changes, please refer to the documentation for more details on this https://vue-i18n.intlify.dev/guide/migration/breaking10.html.

**Comment by slucas-elsa** [comment]
I can't understand what your solution may be : 
 * Force the upgrade to vue-i18n v10 ?
 * stay with the version i'm using now (9.13.1) and I can't use tc with useI18n ?
 * The bug will be fixed when there is a version using vue-i18n v10 ?

**Comment by BobbieGoede** [comment]
Sorry if my previous message was confusing. The `tc` function is only available as legacy API in `vue-i18n` v9, and has been fully replaced with an oveload of `t` in `vue-i18n` v10.

If you're using composition API, that means you likely don't want to use the legacy API, refer to this section to see a mapping of their equivalents: https://vue-i18n.intlify.dev/guide/advanced/composition.html#mapping-between-vuei18n-instance-and-composer-instance and this section on pluralization using the `t` function: https://vue-i18n.intlify.dev/guide/advanced/composition#pluralization.

Basically, this is not a bug, the API has changed. If you're still experiencing issues you will have to provide more information about your project and preferably a minimal reproduction.

EDIT - I just noticed the comment was not by the original author, but the information is applicable to both

**Comment by slucas-elsa** [comment]
Thanks a lot for your detailed answer, I understand now.

**Comment by coremyslo** [comment]
What took me some effort to realize is that when migrating from `$tc` to the new `$t`, you need to swap the 2nd and 3rd arguments:

```
$tc('banana', 100, { n: 'too many' })
```
should become:

```
$t('banana', { n: 'too many' }, 100)
```

However, the two-argument syntax remains unchanged:

```
$tc('apple', 100)
```
becomes:

```
$t('apple', 100)
```
